collectd/files/plugin/collectd_zookeeper.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with this
+#  work for additional information regarding copyright ownership.  The ASF
+#  licenses this file to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""Check Zookeeper Cluster.
+
+Requires ZooKeeper 3.4.0 or greater. The script needs the 'mntr' 4-letter word
+command (patch ZOOKEEPER-744) that was now commited to the trunk. The script
+also works with ZooKeeper 3.3.x but in a limited way.
+"""
+
+import collectd
+import socket
+
+CONFIGS = []
+
+ZK_HOSTS = ["localhost"]
+ZK_PORT = 2181
+ZK_INSTANCE = ""
+COUNTERS = set(["zk_packets_received", "zk_packets_sent"])
+# 4-letter cmds and any expected response
+RUOK_CMD = "ruok"
+IMOK_RESP = "imok"
+MNTR_CMD = "mntr"
+
+
+class ZooKeeperServer(object):
```

Comment:
you probably want to use collectd_base.Base and make it work like the other collectd plugins.